cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/OperatorsTest.scala

```diff
@@ -222,7 +220,8 @@ class OperatorsTest extends DocumentingTest with QueryStatisticsTestSupport {
           |* less than or equal to: `\<=`
           |* greater than or equal to: `>=`
           |* `IS NULL`
-          |* `IS NOT NULL`""".stripMargin)
+          |* `IS NOT NULL`
+          |* matching a regular expression: `=~`""".stripMargin)
```

Comment:
I'd expect this to be added to `String-specific comparison operators comprise` section below.

Reply:
You're right. Thanks for spotting that.